docs/demo.md

```diff
@@ -23,8 +23,8 @@ We have recorded videos on various sections of IPR from the analysts perspective
 
 The demo instance of IPR is at [pori-demo.bcgsc.ca/ipr](https://pori-demo.bcgsc.ca/ipr). This will bring you to the login page of the IPR client application. Use the following credentials to log in to the demo application
 
-- username: `iprdemo`
```

Comment:
we don't plan to update the live demo (afaik) so these should actually stay the same. The users should only change for the instructions on setting it up yourself

Reply:
Ok, so new issue here for that last comment: https://github.com/bcgsc/pori/issues/60
Also, going back to password 'iprdemo' until further notice.